client/src/views/Room.vue

```diff
@@ -292,11 +294,22 @@ export default defineComponent({
 			}
 		}
 
+		const isIframeBasedPlayer = computed(() => {
+			const service = store.state.room.currentSource?.service;
+			return ["youtube", "peertube"].includes(service || "");
+		});
```

Comment:
This feels rather brittle. It's assuming the implementation details of these players. Could be a footgun later.